test/e2e_test.dart

```diff
@@ -242,6 +242,54 @@ void main() {
         expect(buildResult.stderr, isEmpty);
       });
 
+      test('create -t flame_game', () async {
+        final directory = Directory(path.join('.tmp', 'very_good_flame_game'));
+
+        final result = await commandRunner.run(
+          ['create', 'very_good_flame_game', '-t', 'flame_game', '-o', '.tmp'],
+        );
+        expect(result, equals(ExitCode.success.code));
+
+        final formatResult = await Process.run(
+          'flutter',
+          ['format', '--set-exit-if-changed', '.'],
+          workingDirectory: directory.path,
+          runInShell: true,
+        );
+        expect(formatResult.exitCode, equals(ExitCode.success.code));
+        expect(formatResult.stderr, isEmpty);
+
+        final analyzeResult = await Process.run(
+          'flutter',
+          ['analyze', '.'],
+          workingDirectory: directory.path,
+          runInShell: true,
+        );
+        expect(analyzeResult.exitCode, equals(ExitCode.success.code));
+        expect(analyzeResult.stderr, isEmpty);
+        expect(analyzeResult.stdout, contains('No issues found!'));
+
+        final testResult = await Process.run(
+          'flutter',
+          ['test', '--no-pub', '--coverage'],
+          workingDirectory: directory.path,
+          runInShell: true,
+        );
+        expect(testResult.exitCode, equals(ExitCode.success.code));
+        expect(testResult.stderr, isEmpty);
+        expect(testResult.stdout, contains('All tests passed!'));
+
+        final testCoverageResult = await Process.run(
+          'genhtml',
+          ['coverage/lcov.info', '-o', 'coverage'],
+          workingDirectory: directory.path,
+          runInShell: true,
+        );
+        expect(testCoverageResult.exitCode, equals(ExitCode.success.code));
+        expect(testCoverageResult.stderr, isEmpty);
+        expect(testCoverageResult.stdout, contains('lines......: 97.8%'));
```

Comment:
Generated files aren't currently excluded from coverage. Will update this in a subsequent PR